Pages/SpecialEquipment.razor

```diff
@@ -0,0 +1,72 @@
+﻿@page "/specialequipment"
+@using Karma.Models
+@using System.Linq;
+<h1>Special Equipment</h1>
+
+<p>Here you can find the special equipment each volunteer owns.</p>
+
+@if (m_httpContextAccessor.HttpContext.User.Identity.IsAuthenticated)
+{
+    <table class="table">
+        <tbody>
+            <tr>
+                <th>Name</th>
+                <th>Surname</th>
+                <th>Equipment</th>
+            </tr>
+            @foreach (var pair in GetPairsOfVolunteersAndEquipment())
+            {
+                <tr>
+                    <td>@pair.VolunteerName</td>
+                    <td>@pair.VolunteerSurname</td>
+                    <td>
+                        @if (pair.EquipmentList != null)
+                        {
+
+                            @foreach (var equipmentItemName in pair.EquipmentList)
+                            {
+                                <div>
+                                    @equipmentItemName
+                                </div>
+                            }
+                        }
+
+                    </td>
+                </tr>
+            }
+        </tbody>
+    </table>
+}
+else
+{
+    <label for="fname">Please LogIn</label>
+}
+@code
+{
+    private KarmaContext KarmaContext { get; } = new KarmaContext();
+
+    private class VolunteerAndEquipment
```

Comment:
I will move it to the different file and change to struct to meet the requirements in the next PR(if it works)

Reply:
awesome